CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Updated dependency to properly style the hover state for a selected, non-disabled tab. [#41](https://github.com/blackbaud/skyux-theme/pull/41)
```

Comment:
These should include the repo name to be consistent with our SKYUX2 changelog:
`[blackbaud/skyux-theme#41](https://github.com/blackbaud/skyux-theme/pull/41)`

Reply:
Good catch. Fixed